README.md

```diff
@@ -1,204 +1,270 @@
-HISTORY_DB_DATA_EXPIRATION  |Seconds before removing an entry in MongoDB                   |604800
-MONGO_SHARD                 |Activate the use of sharding or not                           |False
-AUTH_URL                    |Auth url address                                              |"http://auth:5000"
 KAFKA_ADDRESS               |Kafta address                                                 |"kafka"
 KAFKA_PORT                  |Kafka port                                                    |9092
-KAFKA_HOSTS                 |KAFKA_ADDRESS concatenated with KAFKA_PORT                    |"kafka:9092"
 KAFKA_GROUP_ID              |Group ID used when creating consumers                         |"history"
-DATA_BROKER_URL             |Data Broker address                                           |"http://data-broker"
-DEVICE_MANAGER_URL          |Device Manager address                                        |"http://device-manager:5000"
 DOJOT_SUBJECT_TENANCY       |Global subject to use when publishing tenancy lifecycle events|"dojot.tenancy"
 DOJOT_SUBJECT_DEVICES       |Global subject to use when receiving device lifecycle events  |"dojot.device-manager.device"
 DOJOT_SUBJECT_DEVICE_DATA   |Global subject to use when receiving data from devices        |"device-data"
 DOJOT_SERVICE_MANAGEMENT    |Global service to use when publishing dojot management events |"dojot-management"
-LOG_LEVEL                   |Define minimum logging level                                  |"INFO"
-PERSISTER_PORT              |Port to be used by persister sevice's endpoints               |8057
 
 
-### How to install the service
+### **How to install Persister service**
 
-### Standalone setup
+#### **Persister service in standalone mode**
 
-It is recommended that all packages are
-installed within a [virtual environment](https://github.com/pypa/virtualenv).
-It can be created by:
+The Persister service is installed in exactly the
+[same way as the History service](#history-service-in-standalone-mode), just
+note that what changes is the _virtual environment_ to be created, in this case,
+the directory name of the _virtual environment_ must refer to the _Persister_
+service. If you want, you can also use the same _virtual environment_ as
+_History_ service, no problem, but be aware of that.
+
+
+#### **Persister service on Docker**
+
+Like the _standalone mode_, the _Persister_ on Docker is identical to
+[that of the History service](#history-service-on-docker),
+paying attention only to the name of the docker image to be generated.
+In this case, it must refer to the Persister service.
 
-```shell
-virtualenv ./venv
-source ./venv/bin/activate
-```
 
-Which will create a `venv` folder in current directory and install
-basic Python packages there. The second command will set all environment
-variables so that new Python packages are installed there as well.
+### **Running the Persister service**
 
```

Comment:
Beforehand, you need an already running dojot instance in your machine. Check out the
[dojot documentation](https://dojotdocs.readthedocs.io)
for more information on installation methods.